### PR TITLE
#409 task 'Artifactory Tools Installer' raises error 401

### PR DESCRIPTION
**Pull request by Shahyads**
- [x] All [tests](https://github.com/jfrog/jfrog-azure-devops-extension#testing) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

**Comment by Shahyads** [comment]
I have read the CLA Document and I hereby sign the CLA

File: jfrog-tasks-utils/utils.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ function createAuthHandlers(serviceConnection) {
     }
 
     // Check if Artifactory should be accessed anonymously.
-    if (artifactoryUser === '') {
+    if (artifactoryUser === '' || artifactoryUser === undefined) {
```
Comment (sverdlov93) [inline jfrog-tasks-utils/utils.js]:
```suggestion
   if (!artifactoryUser) {
```

Comment (sverdlov93) [reply jfrog-tasks-utils/utils.js]: @Shahyads

**Comment by Shahyads** [comment]
#409 task 'Artifactory Tools Installer' raises error 401

**Comment by Shahyads** [comment]
thanks @sverdlov93, I just fixed it

**Review by yahavi — APPROVED** [review]

**Pull request merged**